pydatastructs/linear_data_structures/tests/test_arrays.py

```diff
@@ -127,3 +127,44 @@ def test_DynamicOneDimensionalArray():
     b.append(4)
     b.append(5)
     assert [b[i] for i in range(b.size)] == [1, 2, 3, 4, 5, None, None]
+
+def test_DynamicOneDimensionalArray2():
+    DODA = DynamicOneDimensionalArray
+    root = TreeNode(1, 100)
+    A = DODA(TreeNode, [root])
+    assert str(A[0]) == "(None, 1, 100, None)"
+
+# To Do: Fix this size issue:
+
+# def test_DynamicOneDimensionalArray3():
+#     DODA = DynamicOneDimensionalArray
+#     A = DODA(int, 1)
+#     A.delete(1)
+#     print(A._size)
+#     print(str(A))
+#     A.append(2)
+#     print(str(A))
+#     print(A.size)
+#     A.append(3)
+#     print(str(A))
+#     print(A.size)
```

Comment:
Its causing a bug? So remove the comment from here and open an issue with this test.

Reply:
Opened here: https://github.com/codezonediitj/pydatastructs/issues/557